common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -318,6 +317,8 @@ def load_item(self, location, for_parent=None):  # pylint: disable=method-hidden
                         ).replace(tzinfo=UTC)
                     module._edit_info['published_by'] = raw_metadata.get('published_by')
 
+                module._field_data = OverrideModulestoreFieldData.wrap(module)  # pylint: disable=protected-access
```

Comment:
@rlucioni I think the class and function is doing the right thing, but the name is what causing the confusing. The name of the function is called `wrap` means it will wrap something and return a wrapped instance. Can we get a better name for the `wrap` function here? Like `wrap_field_data_from_module`?